### DISCUSSION TITLE
Missing CITY Code

### DISCUSSION THREAD

**Discussion by abhishek-patel-shl** [dr5hn/countries-states-cities-database#285] [Ideas]
Is it possible to add CITY code as well in the api along with the name?

**Comment by dr5hn** [comment]
Hi @abhishek-patel-shl 

We don't have a CITY code attached to cities in our DB.
https://raw.githubusercontent.com/dr5hn/countries-states-cities-database/master/cities.json

Also it's quite not possible to add one at the moment.
